README.md

```diff
@@ -40,6 +40,8 @@ Usage of ./grpc-rest-proxy:
       --transport.http.server.gracefulTimeout duration     graceful timeout (default 5s)
       --transport.http.server.readHeaderTimeout duration   read header timeout (default 5s)
       --transport.http.server.readTimeout duration         read timeout (default 10s)
+      --service.jsonencoder.emitUnpopulated                emit unpopulated fields
```

Comment:
Add more detailed description for new cli flags. Imagine user who is using this cli app for the first time.